Quaver.Shared/Screens/Gameplay/Rulesets/Keys/GameplayRulesetKeys.cs

```diff
@@ -30,23 +32,26 @@ public class GameplayRulesetKeys : GameplayRuleset
         ///     the playfield and the HitObjectManager.
         ///
         ///     We can't intiialize it in Playfield as that gets created first.
+        ///
+        ///     This is a list because multiple scroll directions require multiple Timing Line Managers.
+        /// 
         /// </summary>
-        public TimingLineManager TimingLineManager { get; }
+        public List<TimingLineManager> TimingLineManager { get; } = new List<TimingLineManager>();
```

Comment:
The timing lines for upscroll & upscroll/split are not correct.

![img](https://i.imgur.com/TdJ4G2B.png)

versus

![img](https://i.imgur.com/K6qgC93.png)

Reply:
Changes made in https://github.com/Quaver/Quaver/pull/477 to avoid cluttering the PR.